third_party/move/move-compiler-v2/tests/checking/positional_fields/named_tuple_ability_decl_ok.move

```diff
@@ -0,0 +1,18 @@
+module 0x42::test {
+    struct S() has copy, key;
+
+    struct S1(u8) has drop;
```

Comment:
Looks like prefix `struct S0 has drop (u8, u8);` is not supported? Is is planned in a followup PR?